src/components/FrontPage.astro

```diff
@@ -16,15 +17,7 @@ import { Blob } from '@components/BgBlob.tsx';
       delay={100}
     />
     <hr class="separator separator--line" />
-    <NavBar
-      client:only="react"
-      links={[
-        { href: '/about', text: 'About' },
-        { href: '/portfolio', text: 'Portfolio' },
-        { href: '/resume', text: 'Resume' },
-        { href: '/contact', text: 'Contact' },
-      ]}
-    />
+    <NavBar client:only="react" links={getPagesLinks({ excludeKey: 'home' })} />
```

Comment:
I set up a shared method `getPagesLinks` to return an array of `links` from the `PagesConfig`. You can optionally pass an `excludeKey` to avoid showing links for the page you are currently on.